# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gsl/k6_k7.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ode-initval/rk8pd.c.orig  Sat Dec  1 15:10:29 2001
+++ ./ode-initval/rk8pd.c       Sat Dec  1 15:13:07 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Prince-Dormand constants */

 static const double Abar[] = {
--- ./ode-initval/rkck.c.orig   Tue Jun 12 13:12:36 2001
+++ ./ode-initval/rkck.c        Sat Dec  1 15:14:35 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Cash-Karp constants */
 static const double ah[] = { 1.0 / 5.0, 0.3, 3.0 / 5.0, 1.0, 7.0 / 8.0 };
 static const double b21 = 1.0 / 5.0;
--- ./ode-initval/rkf45.c.orig  Sat Dec  1 15:39:22 2001
+++ ./ode-initval/rkf45.c       Sat Dec  1 15:40:02 2001
@@ -27,6 +27,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Runge-Kutta-Fehlberg constants */
 static const double ah[] = { 1.0/4.0, 3.0/8.0, 12.0/13.0, 1.0, 1.0/2.0 };

